### ISSUE TITLE
Ablation study for the effect of IPC.

### ISSUE THREAD

**Issue by Aerikervalid** [vefalun/HTFormer#7]
Hi ，Great work！ I would like to ask this error about different joint tests that have relevant code？
Look forward to it and thank the author for your reply！

**Comment by vefalun** [comment]
Yes, in fact, during the process of calculating MPJPE (Mean Per Joint Position Error), we can directly output the error of each joint to obtain the error on each individual joint.

**Comment by Aerikervalid** [comment]
Thank you for your answer! But I still don't know how to output it. Can you point out where the code is located?

**Comment by jianlai123-123** [comment]
Hello, can you successfully add the refine module? If so, can you share how to run it? I look forward to your reply, thank you.@Aerikervalid

**Comment by Aerikervalid** [comment]
> 您好，你能成功添加优化模块吗？如果是这样，你能分享一下如何运行它吗？期待您的回复，谢谢。

Hi, I did not intend to use the refine module, so I have not added it before. But I also want to try using this module recently, it should not be particularly difficult .

**Comment by jianlai123-123** [comment]
Okay, if you can get it working, can I talk to you about it? thanks again.@Aerikervalid
